services/web/server/tests/unit/with_dbs/03/garbage_collector/test_resource_manager.py

```diff
@@ -636,17 +642,18 @@ async def mocked_notification_system(mocker):
 @pytest.mark.parametrize(
     "user_role, expected_save_state",
     [
-        pytest.param(
-            UserRole.GUEST,
-            False,
-            marks=pytest.mark.xfail(
-                reason=(
-                    "GC is wrongly assuming that only 1 tab can be opened for GUEST users. "
-                    "SEE: https://github.com/ITISFoundation/osparc-simcore/issues/3770"
-                )
-                #
-            ),
-        ),
+        # pytest.param(
+        #     UserRole.GUEST,
+        #     False,
+        #     marks=pytest.mark.xfail(
+        #         reason=(
+        #             "GC is wrongly assuming that only 1 tab can be opened for GUEST users. "
+        #             "SEE: https://github.com/ITISFoundation/osparc-simcore/issues/3770"
+        #         )
+        #         #
+        #     ),
+        # ),
```

Comment:
Why was this changed? Form my knowledge the issue is still there.

Reply:
Hey Andrei, I didn't do any "logical" changes, just rearranged/renamed the files. I think it might be connected with the timing that is included in the test `await asyncio.sleep(SERVICE_DELETION_DELAY + 1)` because when i am changing this parameter based on that it ether passes or fails. We can have a look tomorrow together.

Reply:
So this is the correct state of the test. feel free to remove the comment above